pkg/ccl/crosscluster/physical/stream_ingestion_frontier_processor.go

```diff
@@ -439,7 +439,7 @@ func (sf *streamIngestionFrontier) maybeCheckForLaggingNodes() error {
 	// We halve the frequency relative to the ReplanFrequency setting (i.e.
```

Comment:
Comment still refers to `ReplanFrequence`.

Reply:
Fixed. Also remove the halving of the cluster setting, which was old cruft.